app/models/deck.rb

```diff
@@ -0,0 +1,10 @@
+class Deck < ActiveRecord::Base
+  belongs_to :user
+  has_many :cards
+
+  scope :get_current, ->{ where('current = true') }
+
+  def falsify_all_others
+    self.class.where('id != ? and current', self.id).update_all("current=false")
```

Comment:
Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.